tools/tools.go

```diff
@@ -0,0 +1,10 @@
+//+build tools
```

Comment:
Nitpick: Can you add a space between `//` and `+build`? GoLand complains to me if I don't add a space there.